SDWebImage/Core/SDWebImageIndicator.m

```diff
@@ -41,7 +41,7 @@ - (instancetype)init {
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wdeprecated-declarations"
 - (void)commonInit {
-    self.indicatorView = [[UIActivityIndicatorView alloc] initWithActivityIndicatorStyle:UIActivityIndicatorViewStyleWhite];
+    self.indicatorView = [[UIActivityIndicatorView alloc] initWithActivityIndicatorStyle:UIActivityIndicatorViewStyleMedium];
```

Comment:
Should use both compile and runtime check for visionOS/iOS, because this API is iOS 13+

I decided to still maintain SD as iOS 11+ (SwiftUI as iOS 14+)